ttle.py

```diff
@@ -2,18 +2,23 @@
 
 from direct.directnotify.DirectNotifyGlobal import directNotify
 from direct.showbase.ShowBase import ShowBase
+from direct.gui.OnscreenText import OnscreenText
 
 from panda3d.core import loadPrcFile, loadPrcFileData
 
 from tkinter import Tk, messagebox
+from toontown.toonbase import ToontownGlobals
 
 import asyncio
 import argparse
 import builtins
-import json
 import os
 import sys
 
+red = (1, 0, 0, 1)
```

Comment:
This looks bad, just do color = (1.0, 0.0, 0.0, 1.0) in the framerate counter task